backstop/tests/column-filter.html

```diff
@@ -43,14 +43,28 @@ <h2 id="default">Text filter</h2>
       <div class="iui-column-filter">
         <label class="iui-input-container">
           <div class="iui-label">Show items with value that</div>
-          <div class="iui-select">
+          <div class="iui-input-with-icon">
             <div
               tabindex="0"
               id="test-select-1"
               class="iui-select-button"
             >
               <span class="iui-content">Starts with</span>
             </div>
+            <span
+              class="iui-end-icon iui-actionable"
+              aria-hidden="true"
+            >
+              <svg
```

Comment:
I believe we should be able to use a web component icon instead:
```html
<svg-caret-down-small
	class="iui-end-icon iui-actionable"
	aria-hidden="true"
></svg-caret-down-small>
```

You'll need to include this in the `<head>`:
```html
<script
	type="module"
	src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/caret-down-small"
></script>
```

Reply:
Changed icons to web components.